gradle/libs-urbi.versions.toml

```diff
@@ -9,8 +9,8 @@ android-places = "3.3.0"
 androidx-activity = "1.8.1"
 androidx-camera = "1.3.0"
 androidx-camera-view = "1.3.0"
-androidx-compose-bom = "2024.01.00"
-androidx-compose-material3-beta = "1.2.0-rc01"
+androidx-compose-bom = "2024.02.00"
+androidx-compose-material3-beta = "1.2.0"
```

Comment:
We can actually remove this because compose bom `2024.02.00` uses material3 `1.2.0`.

Reply:
you're right